### PR TITLE
DEVX-5909 - Add Audio Streamer to OpenTok .Net SDK 

### PR DESCRIPTION
**Pull request by Tr00d**
Previous PR can be found [here](https://github.com/opentok/Opentok-.NET-SDK/pull/201)

### CODE REVIEW

File: OpenTok/AudioStreamer/ConnectRequest.cs [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+        {
+            ValidateSessionId(sessionId);
+            ValidateToken(token);
+            this.SessionId = sessionId;
+            this.Token = token;
+            this.Socket = socket;
+        }
+
+        /// <summary>
+        ///     The ID of a session (generated with the same `APIKEY` as specified in the URL) where the streams will be gathered
```
Comment (jeffswartz) [inline OpenTok/AudioStreamer/ConnectRequest.cs:49]: Please mimic the descriptions in the REST API documentation: https://tokbox.com/developer/guides/audio-connector/rest/.

Comment (jeffswartz) [reply OpenTok/AudioStreamer/ConnectRequest.cs:49]: Also, please add some info to the README.md file.

**Review by jeffswartz — COMMENTED** [review]
Should we rev the version to 3.10.0 (in the `<Version>` tag in the OpenTok.csproj file)?

File: OpenTok/OpenTok.AudioStreamer.cs [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+        private const string ConnectEndpoint = "connect";
+        private const string ProjectEndpoint = "project";
+        private const string StopEndpoint = "stop";
+
+        /// <summary>
+        /// </summary>
+        /// <param name="apiVersion"></param>
+        /// <param name="request"></param>
+        /// <returns></returns>
+        public async Task<ConnectResponse> ConnectAudioStreamerAsync(string apiVersion, ConnectRequest request)
```
[the remark this answers is not in the document]
Comment (Tr00d) [reply OpenTok/OpenTok.AudioStreamer.cs]: I renamed everything to AudioConnector. When I implemented both calls, it was still AudioStreamer.
Indeed, I initially put AudioConnector-specific objects in their own namespace. A flat hierarchy can quickly become a mess depending on how many endpoints we get. As requested, I removed it anyway.

Regarding the method signature with 3 arguments, I disagree. Primitive obsession is rarely a good idea, especially when associated behaviours such as Validation or data preparation exist. Here's the perfect opportunity to use a value object. This code is specific to this request and shouldn't OpenTok.cs responsibility.

**Comment by Tr00d** [comment]
> Should we rev the version to 3.10.0 (in the `<Version>` tag in the OpenTok.csproj file)?

We don't have to bump the version manually. It's done automatically during the release process, based on the GitHub release.

**Review by jeffswartz — APPROVED** [review]

**Review by jeffswartz — APPROVED** [review]

**Pull request merged**